### PR TITLE
Fix/supress ReSharper warnings in providers

### PR DESCRIPTION
**Pull request by AndriySvyryd**
@lajones

### CODE REVIEW

File: src/Microsoft.EntityFrameworkCore.InMemory/Extensions/Internal/InMemoryLoggerExtensions.cs [hunk 1]
```diff
@@ -5,7 +5,8 @@
 using Microsoft.EntityFrameworkCore.Infrastructure;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.EntityFrameworkCore.Internal
+// ReSharper disable once CheckNamespace
+namespace Microsoft.EntityFrameworkCore
```
Comment (lajones) [inline src/Microsoft.EntityFrameworkCore.InMemory/Extensions/Internal/InMemoryLoggerExtensions.cs]: Are you sure on this? It's in the `Internal` directory?

Comment (AndriySvyryd) [inline src/Microsoft.EntityFrameworkCore.InMemory/Extensions/Internal/InMemoryLoggerExtensions.cs]: In this case the class itself is internal

Comment (lajones) [inline src/Microsoft.EntityFrameworkCore.InMemory/Extensions/Internal/InMemoryLoggerExtensions.cs]: Hmmm. OK.

**Comment by lajones** [comment]
:shipit:

**Pull request merged**